docs/TUTORIAL.md

```diff
@@ -41,160 +47,307 @@ This tutorial uses the following billable components of Google Cloud:
     ```
     dbt test --profiles-dir profiles/ -s test_type:singular
     ```
 
-1. Generate the documentation for the project:
+
+9. Generate the documentation for the project:
     ```
     dbt docs generate --profiles-dir profiles && dbt docs serve --profiles-dir profiles 
     ```
 
-1. Explore the lineage of the models, and their detailed documentation. You see that  the documentation includes all the models’ documentation as specified in the  models/schema.yml files, and all the code of the models.
+
+10. In the output that you see, search for, and then click, the following URL text: http://127.0.0.1:8080
+    
+    Your browser opens a new tab that shows the dbt documentation web interface.
+
+
+11. Explore the lineage of the models, and their detailed documentation. You see that  the documentation includes all 
+the models’ documentation as specified in the  models/schema.yml files, and all the code of the models.
 
 ----
 ## (Optional) Containerize the transformations
 1. Create a container for the BigQuery data load step, and push the container to Google Container Repository:
     ```
-    cd ../components/data_load/     # Note the Dockerfile in this folder
+    cd ../data_load/     # Note the Dockerfile in this folder
     gcloud builds submit --tag $GCR_DATALOAD_IMAGE  # Pushes the image to GCR     
     ```
     The Dockerfile in this directory enables this containerization, which simplifies orchestration of the workflow.
 
-1. Containerize the DBT code for the data transformation step, and push the container to Google Container Repository. 
+
+2. Containerize the DBT code for the data transformation step, and push the container to Google Container Repository. 
     ```
-    cd ../../dbt
+    cd ../dbt
     gcloud builds submit --tag $GCR_DBT_SAMPLE_REPORTING_IMAGE
     ```
     Containerization helps you to create a package that can be easily versioned and deployed. 
 
+
+3. Retrieve the path of the Airflow UI and the GCS bucket for dags. 
+   You can use the terraform output command to retrieve those variables.
+    ```
+    cd ../../../../common_components/orchestration/infrastructure/ 
+    terraform output | grep airflow
+    ```
+    For convenience, you may want to store the output in environment variables.
+    ```
+    AIRFLOW_DAG_GCS=$(terraform output --raw airflow_dag_gcs_prefix)
+    AIRFLOW_UI=$(terraform output --raw airflow_uri)
+    ```
+   You will need to use the AIRFLOW_DAG_GCS to upload the Airflow DAG, and the AIRFLOW_UI to 
+    log into Composer and see the progress of the orchestration.
+
+
+5. Upload the home loan delinquency dag.
+    ```
+    cd ../../../use_cases/examples/home_loan_delinquency/deploy/
+    gsutil cp run_dag.py $AIRFLOW_DAG_GCS
+    ```
+   
+6. Head to the Airflow UI and see the process executing. 
+   Execute the command below to retrieve the UI, and click on the link:
+   ```
+   echo $AIRFLOW_UI
+   ```
+    
+   
+8. In the UI, you should see a diagram like the one below.
```

Comment:
Except you see instead the top-level view where you choose the dag.

You also may need to wait a minute or two, and then you can click on the last run.